_posts/2019-08-20-service-objects.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Service Objects: beyond fat models and skinny controllers"
+date: 2019-09-03 10:00:00
```

Comment:
@cleicar can you update the data here and in the name of the file? we can publish it today  🤙